tests/testthat/test-git.R

```diff
@@ -11,30 +11,30 @@ test_that("git parent directory can be located", {
   path <- tempdir()
   expect_error(git_root_directory(path), sprintf(
     "Can't run import as can't locate git directory from path %s. %s", path,
-    "Import must be under version control to be run."))
+    "Import must be under version control to be run."), fixed = TRUE)
 
 })
 
 test_that("git status can be retrieved and clean can be checked", {
   path <- build_git_demo()
-  expect_equal(git_status(path),
-               list(success = TRUE, code = 0,
-                    output = character(0), clean = TRUE))
+#  expect_equal(git_status(path),
```

Comment:
I think we can remove the tests of `git_status`. This is no longer a function we implement (or want to implement in the future) so I don't see much reason to keep them around.

Reply:
Great - thought I'd comment them and give you a look before I obliterated them. Done